draft-rwbr-tsvwg-signaling-use-cases.md

```diff
@@ -375,6 +379,29 @@ than audio.
 
 Todo: this section on cooperation needs editing.
 
+### Live Streaming {#example-live-straming}
+
+Live streaming is differnet from video streaming given the focus is on reducing latency and keeping the streaming as close to the live feed. In this scenario, video might be more prioirty than audio in some cases (e.g., Super Bowl, Olympics or the World Cup) while audio has more priority (e.g., music concert). This also prioritizes latency (to a certain extent) over quality. Network-to-host information proves vital in selecting of the quality to minimize the latency. Signaling of metadat could reduce the latency and improve performance of the live streaming stated in {{MOQ-PRIORITY}}.
+
```

Comment:
I don't think the use case and requirements draft should about discuss about metadata attributes defined in another spec.

Reply:
We are mentioning the same video key and delta frames in the below table Tiru. Just referencing Ali's presentation to bolster that claim. Do you think I'll remove the reference?

Reply:
To @tireddy2's point, I am of the mind to pull all the details of metadata specifics from this I-D.  This I-D is trying to concentrate on justification for the signaling protocol itself, not what it would (or could) contain.  Specifically, what *breaks* or *would be impossible* without a signaling protocol should be highlighted.  I realize the document is not perfect at highlighting those breakages/impossibilities quite yet.

Reply:
I agree. My train of thought here: Are we planning to make the justification and then provide an example of how signaling would solve/help solve the problem that we are representing? I wonder if it will be clear enough if we just focus on the signaling without stating anything specific, given we are listing practical applications to bolster our claim for the need of signaling protocol. If we are doing that, then, might as well use the other draft as a reference of how the signaling is both extendable and protocol/career agnostic.

Reply:
The tables of which signals we would send fail to show how a separate signaling channel is useful -- as those metadata bits could be carried in IPID, UDP Options, IPv6 flow id, IPv6 hop-by-hop extensions, and similar.  That is, without a (separate) signaling protocol.

Reply:
> fail to show how a separate signaling channel is useful

which I think is fine for the moment as I suspect that there is still an effort needed to motivate why sharing data is useful.

Please note also that we have this text in the draft:

`This document does not intend to define any signaling protocol nor call whether a new signaling protocol, a new extension, one or more signaling protocols are needed.`